world/eqemu_api_world_data_service.cpp

```diff
@@ -270,8 +272,64 @@ void EQEmuApiWorldDataService::get(Json::Value &r, const std::vector<std::string
 	}
 	if (m == "get_reload_types") {
 		getReloadTypes(r);
-	}
+	}				
```

Comment:
Weird whitespace here.

Reply:
I don't think the white space is weird.  Looks quite normal really.  Just seems to be oddly located.   Ok, so that is my attempt at white space humor.  haha

Updated.

Reply:
> I don't think the white space is weird. Looks quite normal really. Just seems to be oddly located. Ok, so that is my attempt at white space humor. haha
> 
> Updated.

😆